assets/templates/signup.html

```diff
@@ -51,9 +51,15 @@
         <header>
           <h1 class="logo-doppler-new">Doppler</h1>
           <div class="language-selector">
-            <div class="option" id="spanish-selector">
-              <span><img src="img/spanish-flag.svg"><strong>ES</strong></span>
-              <a class="option--item" href="#"><img src="img/english-flag.svg"><strong>EN</strong></a>
+            <div class="lang-option option--selector">
+              <button class="lang--es">
+                ES
+              </button>
+            </div>
+            <div class="lang-option">
+              <button class="lang--en">
```

Comment:
Need this in `<a>` because is a link

Reply:
Good catch @fcoronelmakingsense! 

@damianmuti what do you think about fixing it and also applying the related changes in webapp before merging it?

Reply:
Sure, @andresmoschini ! I've just pushed the changes to this PR 👌

Reply:
Fo, @fcoronelmakingsense, feel free to ignore these changes in your tasks related to signup page, forgot page, etc.